code/__DEFINES/is_helpers.dm

```diff
@@ -18,7 +18,7 @@ GLOBAL_VAR_INIT(magic_appearance_detecting_image, new /image) // appearances are
 
 // The filters list has the same ref type id as a filter, but isnt one and also isnt a list, so we have to check if the thing has Cut() instead
 GLOBAL_VAR_INIT(refid_filter, TYPEID(filter(type="angular_blur")))
-#define isfilter(thing) (!hascall(thing, "Cut") && TYPEID(thing) == GLOB.refid_filter)
+#define isfilter(thing) (hascall(thing, "Cut") && TYPEID(thing) == GLOB.refid_filter)
```

Comment:
Note that `/filters` actually have "Cut" call.

Reply:
Is this meant to check for /filter or /filters, since by the sound of it it is actually checking for a filter, not the filters list. Does anything use this though?

Reply:
By the cursed byond logic, /filters starts from the all same reference id (0x53~), and has the same proc call.
There is no way to identify if your filters is the mother of all filters in your atom, or individual `/filters` (Yes, singular is `/filters`. Thanks Byond.)